s3/src/main/resources/reference.conf

```diff
@@ -14,33 +14,22 @@ akka.stream.alpakka.s3 {
     secure = true
   }
 
-  # default values for AWS configuration. If credentials and/or region are not specified when creating S3Client,
-  # these values will be used.
+  # default values for AWS configuration
   aws {
-    # DEPRECATION WARNING
-    # Support for settins directly on this level is left for compatibility.
-    # It may be removed in future releases.
-    # An attempt will be made to read these if no `aws.credentials.*` params
-    # will be provided. If you used these parameters, switch to the new format.
-    # access-key-id
-    # secret-access-key
-    # Equivalent config would be:
-    # akka.stream.alpakka.s3.aws.credentials {
-    #   provider = static
-    #   access-key-id = ${old-access-key-id}
-    #   secret-access-key = ${old-secret-access-key}
-    # }
     # If this section is absent, the fallback behavior is to use the
```

Comment:
This comment describes what is a fallback behavior if complete `credentials { ... }` section is missing. I have changed it to refer to `provider = default` instead of repeating the same.